File: src/python/core/domain/entities/language.py
from dataclasses import dataclass
from typing import Optional
from dataclasses_json import dataclass_json

from ..enums.language_proficiency import LanguageProficiency as LanguageProficiencyEnum


@dataclass_json
@dataclass
class Language:
    Language: Optional[str] = None
    Proficiency: Optional[LanguageProficiencyEnum] = None
    Evidence: Optional[str] = None

File: src/dotnet/Core/Domain/Entities/Language.cs
using Rag.Candidates.Core.Domain.Enums;

namespace Rag.Candidates.Core.Domain.Entities;

public class Language
{
    public string? Name { get; set; }
    public LanguageProficiency? Proficiency { get; set; }
    public string? Evidence { get; set; }
}